subt/zmq-subt-x2.json

```diff
@@ -71,19 +65,15 @@
               ["rosmsg.cmd", "transmitter.raw"],
 
               ["rosmsg.rot", "app.rot"],
-              ["rosmsg.rot", "depth2scan.rot"],
               ["rosmsg.acc", "app.acc"],
               ["rosmsg.orientation", "app.orientation"],
 
               ["rosmsg.scan", "detector.scan"],
-              ["rosmsg.scan", "depth2scan.scan"],
 
               ["rosmsg.image", "detector.image"],
               ["rosmsg.image", "app.image"],
 
-              ["rosmsg.depth", "depth2scan.depth"],
-              ["depth2scan.scan", "app.scan"],
-              ["rosmsg.depth", "detector.depth"],
+              ["rosmsg.scan", "app.scan"],
```

Comment:
I think we need to be able to use both - x2 and x4 robots. The file `zmq-subt-x2.json` is for x2.

Reply:
I agree. Is it a good solution to create a new json config in this directory e.g. zmq-subt-x4.json?

Reply:
yes